### PR TITLE
Add missing appid in URL

### PR DESCRIPTION
**Pull request by mxjeff**
Add missing ``appid`` in http request and rename env to avoid collision with potential other api key.

### CODE REVIEW

**Comment by sumpfralle** [comment]
Thank you for contributing to this plugin!

Sadly the previous usage of the `apikey` (that still remained in the documentation header) got lost recently (in January) in another pull request. Thus your submission is very welcome!

I have two additional thoughts / questions:

1. Maybe you would like to stick with the original name of the environment variable name `apikey`. Otherwise you would need to update the documentation header.
2. Maybe you could combine the two `query_string=...` lines into a single one for better readability? (optional)

Thank you!

**Comment by mxjeff** [comment]
I made some others changes (mainly switch to metric unit system to get rid of kelvin offset).

But in the mean time I noticed there is an issue drawing the main graph.  
For some reason munin is looking for the wrong rdd file. I still have to figure out what's wrong (not even sure the plugin is responsible).

Somebody else testing the plugin would be very welcome :)

File: plugins/weather/openweather_ [hunk 1]
```diff
@@ -37,33 +35,30 @@ then
...
-		temp_max.cdef $KELVIN_BIAS,-
 
 		multigraph $plugin_name.humidity
 		graph_title Humidity in ${CITY}
 		graph_vlabel %
 		graph_category sensors
+		graph_args --upper-limit 100 --lower-limit 0
 		graph_info This graph show the humidity in ${CITY}
-		humidity.label humidity
+		humi.label humidity
```
Comment (sumpfralle) [inline plugins/weather/openweather_]: Please use an easily understandable (but still distinct) fieldname. Mabye `field_humidity`?

Comment (mxjeff) [inline plugins/weather/openweather_]: Agreed, idem with pressure field.

**Comment by mxjeff** [comment]
Indeed, kelvin is a better choice. I followed existing comments without thinking about it.

Having some plugin conf to choose unit between metric, imperial and defaulting to kelvin would be even better (but it deserves another pull request imho).

**Comment by mxjeff** [comment]
I did change back to kelvin+cdef but the [previous cdef config](https://github.com/munin-monitoring/contrib/blob/f2de5497bdcd94604048c4492739f6d83b55b470/plugins/weather/openweather_#L40) was triggering a ``rpn expressions without DEF or CDEF variables are not supported``. I'm not fluent enough in rdd, I let you judge my changes.

My settings: munin 2.0.25, rddtool  1.4.8,  Debian 8.8

File: plugins/weather/openweather_ [hunk 2]
```diff
@@ -37,33 +36,34 @@ then
...
+		temp_avg.cdef temp_avg,$KELVIN_BIAS,-
 
 		multigraph $plugin_name.temp
 		graph_title Temperature in ${CITY}
 		graph_vlabel Celsius
 		graph_category sensors
 		graph_info This graph show the temperature in ${CITY}
 		temp_avg.label avg
-		temp_avg.cdef $KELVIN_BIAS,-
+		temp_avg.cdef temp_avg,$KELVIN_BIAS,-
```
Comment (sumpfralle) [inline plugins/weather/openweather_]: I guess, the original cdef was ok? (`temp_avg.cdef $KELVIN_BIAS,-`)

**Comment by sumpfralle** [comment]
Ping?

**Comment by sumpfralle** [comment]
@mxjeff: any news regarding your proposal?

File: plugins/weather/openweather_ (hunk 2 above)
Comment (mxjeff) [reply plugins/weather/openweather_]: Hi,

I still have an error trying to graph : ``rpn expressions without DEF or CDEF variables are not supported``

I got it solved adding ``temp_avg``.

Setup:
  * Debian 9.4 vanilla
  * munin 2.0.33-1
  * rddtool 1.6.0-1

How could I debug further?
I'll try to reproduce with another set up from scratch.

Cheers and sorry for my delayed answer.

Comment (sumpfralle) [reply plugins/weather/openweather_]: After taking at a look at the `cdef` again, I think, your change was always on the right track - at least I cannot think of a good way, how the previous `cdef` was interpretet. Probably the `cdef` parser in munin was a bit more sloppy in an older release?

Anyway: if you are happy with the current state, I will merge it.
What do you think?

Comment (mxjeff) [reply plugins/weather/openweather_]: I'm ok with it.

I am not familiar with github workflow though.
Should I rebase my branch on top of master or is it ok like that ?

Comment (sumpfralle) [reply plugins/weather/openweather_]: Your changes still do not collide with the current master - thus they can be applied without rebase.

Thank you!

**Pull request merged**